roosevelt.js

```diff
@@ -85,6 +87,17 @@ module.exports = function(params) {
   app.httpServer = httpServer;
   app.httpsServer = httpsServer;
 
+  // add open sockets to array when connected
+  httpServer.on('connection', function (socket) {
+    var socketId = nextSocketId++;
```

Comment:
Do we need the extra var? Could we just make the below line be `sockets[nextSocketId++] = socket;` Since it is defined above across the whole file?